app/app.rb

```diff
@@ -37,6 +37,15 @@
   devotionals.to_json
 end
 
+post '/prayer_list' do
```

Comment:
This route is for `/prayer_list` but it doesn't create a PrayerList record, so it's a little bit surprising. Would this make more sense as a POST to `/user/residents`, instead?

Reply:
Gotcha, so I'm pretty new to correctly naming routes. I've really only used this nested structure when using params. It definitely makes alot more sense though because we are creating the users residents. Do most request names refer to the class they are utilizing? Does that mean I should change my get /home route to get /verse?

Reply:
This is a dense topic, but generally for something to be REST-ful, you'd name your routes according to a certain convention. Usually APIs fall short of fully adhering to REST, but it's a good guideline.
For this app, I don't think we need to go full on REST, so I don't think we need to change /home, since more stuff is probably going to get added to that response, but I am making assumptions here.

For what it's worth, my comment was more of a question to get you thinking about this. We can go with whatever name makes sense, but we should try to be consistent so that we don't have to think too hard.

For reference on REST: https://restfulapi.net